src/pages/api/index.md

```diff
@@ -420,17 +420,46 @@ The name of the object or function you wish to export.
 ## log.level([entry, ..])
 The `log.x` method set provides the ability to log.
```

Comment:
The `log.level` method set provides the ability to log directly off the seneca instance. By default we include a logging adaptor that emits json based logs. You can swap out this adaptor for others; see [Logging with Seneca][] for more information on adaptors